ray-operator/controllers/ray/raycluster_controller_test.go

```diff
@@ -534,6 +551,17 @@ var _ = Context("Inside the default namespace", func() {
 			Eventually(
 				getClusterState(ctx, namespace, rayCluster.Name),
 				time.Second*3, time.Millisecond*500).Should(Equal(rayv1.Suspended))
+
```

Comment:
Could you check that both `suspending` and `suspended` should be false when `suspend` transitions from true to false?

Reply:
This is checked by the new `findRayClusterSuspendStatus` helper in tests.